pom.xml

```diff
@@ -263,7 +263,7 @@
                     </executions>
                     <configuration>
                         <forkCount>${failsafe.forkCount}</forkCount>
-                        <argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
+                        <argLine>-Xmx1024m -XX:MaxPermSize=256m -Duser.country=en -Duser.language=US</argLine>
```

Comment:
we actually have this one to do the same thing. https://github.com/vaadin/flow-components/blob/master/.mvn/jvm.config